demo/genome_config.yaml

```diff
@@ -0,0 +1,206 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            genome--grch37: "TCRBOA7-N-WGS"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data1/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           # Copied from vcf2maf
+            grch37: " reference/genomes/grch37/downloads/hg19-grch37 "     # Copied from vcf2maf
+    
+    hmftools:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data1/{sample_id}.bam"
+            sample_bai: "data1/{sample_id}.bam.bai"
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} 
+            # Note: Only outputs of running Strelka in matched normal mode are accepted
+            # Must be in bgzip with `.vcf.gz` extension. 
```

Comment:
From my naive perspective I am not sure what "must be in gzip" means. I know gzip is for compression, not sure what is meant by "in" or what requires it. Please clarify this comment.

Reply:
It means that file should be compressed in gzip, for it to be used as input for the module.

Reply:
"File must be compressed in gzip, for it to be used as input for the module." would be a clearer comment.
Although ideally we would have something to check the input for us (a test) and also an clear error message output if the attempted input file is not formatted correctly (for example, not compressed with gzip).

Reply:
Good point. Clarification: bgzip is a requirement for these and gz compression will not produce compatible files. The annoying decision of the people who implemented bgzip was to keep a "gz" extension thus making this ambiguous. I agree wholeheartedly with the suggestion that such requirements would ideally be checked to ensure files satisfy them. That's something we can discuss for down the road.